### ISSUE TITLE
comment(email=True) does not work

### ISSUE THREAD

**Issue by kparal** [fedora-infra/bodhi#289]
See the docstring for `comment(email=)`:

```
    def comment(self, update, comment, karma=0, email=None):
        """ Add a comment to an update.

        :arg update: The title of the update comment on.
        :arg comment: The text of the comment.
        :kwarg karma: The karma of this comment (-1, 0, 1)
        :kwarg email: Whether or not to trigger email notifications

        """
```

If I call `bodhi_client.comment(bodhi_update['title'], comment, karma)`, everything works, comment is added. But if I call `bodhi_client.comment(bodhi_update['title'], comment, karma, email=True)`, I get `BodhiClientException: Invalid email address`.

However, it is not supposed to be an email address, it is supposed to be a boolean flag.

python-fedora 0.5.5

**Comment by ralphbean** [comment]
Ha!  Yeah, the docstring must just be wrong.  That ``email`` field is there (I think) to allow anonymous comments.

**Comment by kparal** [comment]
In that case is it still possible to decide whether to send out emails for the submitted comment or not? That's how it used to work in Bodhi1 API (the `email` field was a boolean flag). It's not a critical functionality for us (we want to deprecate taskotron comments soon), but if it is still available in some form, we would make use of it.

**Comment by kparal** [comment]
Oh, and if the feature is no longer available, what is the current behavior? Send emails for every comment? Or has it all moved to FMN already?

**Comment by kparal** [comment]
After talking to Luke yesterday, the `email` attribute now has a different meaning than in Bodhi1 (not sure if he wants to revert it or not), and every submitted comment should trigger an email (maybe even 2, one direct and one through FMN). Either way, it should be OK for taskotron to simply submit comments without using `email=` and the maintainers should get notified.

**Comment by ryanlerch** [comment]
@ralphbean is right, if you supply an email address here, the comment is marked as anon:

https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/services/comments.py#L213

i'll make a quick patch to fix the docstring

**Comment by bowlofeggs** [comment]
The patch for this issue should be included in the upcoming 2.10.0 release:

https://github.com/fedora-infra/bodhi/pull/1744
